packages/data-models/src/language.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * Constants that define a macrolanguage.
+ *
+ * @enum {string} */
+export const Lang = {
+  LATIN: 'lat',
+  GREEK: 'grc',
+  ARABIC: 'ara',
+  PERSIAN: 'per',
+  GEEZ: 'gez',
+  CHINESE: 'zho',
+  SYRIAC: 'syr'
+}
+
+/**
+ * A value object that represents the notion of a language.
+ */
+export default class Language {
+  /**
+   * Creates an instance of a language class.
+   *
+   * @param {Lang} code - A constant that specifies a language.
+   */
+  constructor (code) {
+    if (!code) {
+      throw new Error('Language object cannot be create without a language _code')
+    }
+
+    /**
+     An ISO 639-3 _code.
+     *
+     @private
+     @type {string}
+     */
+    this._code = code
+  }
+
+  static get LATIN () {
+    return new Language(Lang.LATIN)
```

Comment:
A static getter that will create an instance of a `Language` value object for a specified language.